java/src/apps/AppsBundle_es.properties

```diff
@@ -10,7 +10,7 @@ MenuItemSave = Guardar
 MenuItemSaveAs = Guardar como...
 MenuItemQuit = Salir
 MenuPrintDecoderDefinitions = Imprimir definiciones de decodificador...
-MenuPrintPreviewDecoderDefinitions = vista previa de impresión de definiciones del decodificador...
+MenuPrintPreviewDecoderDefinitions = vista previa de impresi\u00f3n de definiciones del decodificador...
```

Comment:
Why these changes? I think the properties files are allowed to use UTF8 characters now.

Reply:
Funny/strange I do not see that change to line 13 in my branch. It might be some behind the scenes IDE job?

Reply:
Do you edit the file in the IDE? If so, the IDE will probably save the file encoded using ASCII characters instead of UTF-8 characters. Before, this was needed since the property files didn't support UTF-8 characters, but they do now. So it would be good if you could change the setting in the IDE to save the property files as UTF-8.

Reply:
I use IntelliJ (works great!). On the subject of resource bundles/properties file it remarks:

"The standard Java API is designed to use the ISO 8859-1 encoding for properties files. You can use escape sequences for characters that are not defined by this encoding. Alternatively, you can define the default encoding for properties files on the project level and use a different API that can read properties files in the encoding you have defined."

Are you sure we made that change in JMRI?